kernel/generic/src/ipc/ipc.c

```diff
@@ -891,6 +883,8 @@ void ipc_init(void)
 	    NULL, 0);
 	answerbox_cache = slab_cache_create("answerbox_t", sizeof(answerbox_t),
 	    0, NULL, NULL, 0);
+	irq_cache = slab_cache_create("irq_t", sizeof(irq_kobject_t),
+	    0, NULL, NULL, 0);
```

Comment:
Hm, so irq_cache is now a cache of irq_kobject_t objects. Wasn't it the case that before there were also pure irq_t objects not visible by userspace and therefore not referencable by a capability?

Reply:
Yeah, they still exist. They just never used this slab for allocations. I think they are all just static.

Reply:
I admit it's mildly confusing, I should probably rename it.